src/components/App/AppStore.ts

```diff
@@ -17,6 +18,11 @@ class AppStore {
     return this.resourceId.indexOf('arn:aws:lambda') === 0;
   }
 
+  @computed get getResourceType() {
```

Comment:
Adding dependency to `Viewer` package in a `App` level class?

Reply:
Maybe AppStore is not a proper naming?
It's actually  viewer/editor store.

On the other hand,   ResourceType is not only a viewer level concept. which should be moved to the level higher than viewer/editor. which should be app level.  Happy to do that.